.github/workflows/docs-build.yml

```
@@ -0,0 +1,39 @@
+    paths:
+      - ".github/workflows/docs-build.yml"
+      - "docs/**"
+    branches:
+      - main
+
+jobs:
+  docs:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)